content/en/docs/contribute/style/style-guide.md

```diff
@@ -116,15 +113,15 @@ The copy is called a "fork". | The copy is called a "fork."
 
 ## Inline code formatting
 
-### Use code style for inline code and commands
+### Use code style for inline code, commands, and API objects
```

Comment:
Adding code styling to API objects would match GitHub's and Google's API conventions, which seems like a trivial way to ease friction. 👍

Reply:
@cmastr, would you be willing to open an issue that recommends a change to use code styling for API objects?
If you do, you can link this PR to the issue.

The reason I'm suggesting that switch is because, firstly, I want to chip in on the discussion with some comments about implemention details, and secondly, I want to separate the “should we do this?” discussion (the issue) from the “is this a good way to do it?” discussion (which can be this pull request and potentially other pull requests if a different person suggests another approach).

All of this is in aid of a low-friction route to revising and improving what seems like a contentious part of the style guide.

Reply:
Yes sure. Let me see if I can figure out how to open an issue and describe it properly.

Reply:
> The reason I'm suggesting that switch is because, firstly, I want to chip in on the discussion with some comments about implemention details, and secondly, I want to separate the “should we do this?” discussion (the issue) from the “is this a good way to do it?” discussion (which can be this pull request and potentially other pull requests if a different person suggests another approach).
> 
> All of this is in aid of a low-friction route to revising and improving what seems like a contentious part of the style guide.

Yes, create an issue to outline any proposed changes.

Reply:
Here you go: #23527

Reply:
Hi. I probably would hold off on changing the heading.
I'd like more information about styling API objects before agreeing to this change (provide links to other
style guides, especially guides that reference Kubernetes concepts).

Reply:
@kbhawkey Google and Github style API objects in monospace font. I think the heading is fine and shouldn't be a blocker.